.dockerignore

```diff
@@ -0,0 +1,3 @@
+Dockerfile
```

Comment:
This is great but just FYI you could use a little regex to condense this down to one line which is something I do often:

`[Dd]ocker*`

This will match upper *or* lower case `D` followed by all lower case `ocker` and any number of characters after that with the `*`.